pkg/eventing/utils.go

```diff
@@ -14,6 +14,10 @@ import (
 
 const publisherProxySuffix = "publisher-proxy"
 
+func GetEPPDeploymentName(eventing v1alpha1.Eventing) string {
```

Comment:
Why pass the whole `eventing v1alpha1.Eventing` instance? Internally the function only needs the name.

Reply:
I just followed Faizan's approach. He does this for the other resources services, roles, etc. Can we leave for consistency reasons?